### ISSUE TITLE
Front photo disappears before send

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#448]
I was in offline mode yesterday in the subway, and tried to add a can of beer I found, as a demo.
- Took front,
- Took nutrition
- Tried to hit send. Popup: you're lacking an image.
- Switch back to nutrition. The front pic had disappeared, had to retake it before sending

**Comment by teolemon** [comment]
```04-22 10:27:53.734 7652-7652/openfoodfacts.github.scrachx.openfood W/PathParser: Points are too far apart 4.000000596046461
04-22 10:27:54.100 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
04-22 10:27:54.111 7652-7659/openfoodfacts.github.scrachx.openfood W/art: Suspending all threads took: 9.356ms
04-22 10:27:54.111 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity.onPhotoReturned(SaveProductOfflineActivity.java:307)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity.-wrap0(SaveProductOfflineActivity.java)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity$1.onImagesPicked(SaveProductOfflineActivity.java:292)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at pl.aprilapps.easyphotopicker.EasyImage.onPictureReturnedFromCamera(EasyImage.java:385)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at pl.aprilapps.easyphotopicker.EasyImage.handleActivityResult(EasyImage.java:287)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at openfoodfacts.github.scrachx.openfood.views.SaveProductOfflineActivity.onActivityResult(SaveProductOfflineActivity.java:284)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.Activity.dispatchActivityResult(Activity.java:6456)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.ActivityThread.deliverResults(ActivityThread.java:3723)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3117)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3162)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2509)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4105)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.ActivityThread.-wrap15(ActivityThread.java)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1373)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:102)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.os.Looper.loop(Looper.java:148)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:5461)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
04-22 10:27:54.112 7652-7652/openfoodfacts.github.scrachx.openfood W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
04-22 10:27:54.155 7652-7652/openfoodfacts.github.scrachx.openfood W/PathParser: Points are too far apart 4.000000596046461
```

**Comment by huzaifaiftikhar** [comment]
Closed in #1712
